src/components/shared/form_fields/RadioField.vue

```diff
@@ -16,6 +16,7 @@
 				@update:modelValue="onFieldChange"
 			>
 				{{ option.label }}
+				<slot :name="`message-${option.value}`"/>
```

Comment:
```suggestion
				<slot :name="`message-${option.value.replace(' ', '-')}`"/>
```
would take care of values with spaces

Reply:
option.value is something like BEZ, SUB, UEB, so I'd like to be naive and say this might be fine